### DISCUSSION TITLE
关于使用场景的想法

### DISCUSSION THREAD

**Discussion by vickingwu** [thinkingjimmy/Learning-Prompt#46] [Ideas]
由AI学英文联想到一个场景，不知道技术上是否可行？
训练一个专属模型，赋予他自己的思想和人格，让远在他乡的家人也能通过这个相同的模型进行互动
好处是，不用考虑时间和空间的限制，相当于人与人实际交互的时间是同步的，但信息输入的时间是异步的

**Comment by thinkingjimmy** [comment]
这个问题，跟很多所谓的为逝世的人训练模型类似。
我认为理论上可行，但实际不行。
因为模型训练所需的数据，一般都是公开的数据，比如家人在微信说过的话。但人的「心里话」是不会说出来的，你无法将这些心里话给模型训练。

你可以拿自己作为例子，将你所有的数据都放到模型里训练可行吗？你的所有数据会包含你的所有心里话吗？

**Reply by vickingwu** [reply]
无法穷举所有场景的每个反应，也很难预测在未知情况下的真实想法，人是感性动物无法完全用语言来tag每个行为和思维
但如果将曾发生过的场景，或能预测的反应提前输入来训练模型，可能会尽量趋近于自己的性格
这种拟人的应用目前已经有了，如character.ai，不过的确无法取代真实的我们

**Reply by wpv-chan** [reply]
我觉得实际上是可以的，LLM的输出方式跟人应该还是有区别的，我们可能会觉得需要知道人心里想什么才能真正模仿这个人说话，但我觉得计算机不需要，计算机也不需要知道人脑是怎么工作的，但是它可以输出跟人类思考结果相似的内容。用公开的数据进行训练，应该就可以实现输出与公开数据高度相似的内容，LLM它不关心你的人格和思想是怎么相互作用得到这些结果的，它只关注寻找合适的概率组合得到这个结果。

**Reply by thinkingjimmy** [reply]
相似可能可以，但只能是公众看到的那一面「相似」。人都是有两面的，模型没有另一面的数据，并且不关心人格的情况和思想下，你问虚拟的张继科你会赌博吗？他只会回答：赌博是违法的。
